src/components/ContactCard/ContactForm/fieldsConfig.js

```diff
@@ -55,6 +70,11 @@ export const fields = [
     type: 'date',
     labelProps: { shrink: true }
   },
+  {
+    name: 'birthcity',
```

Comment:
We should use birth place https://www.rfc-editor.org/rfc/rfc6474.html#section-2.1